src/com_tjnotifications/admin/models/notifications.php

```diff
@@ -31,8 +31,9 @@ public function __construct($config = array())
 				'id',
 				'client',
 				'key',
-				'state', 'a.state'
-
+				'state',
+				'a.state',
```

Comment:
seems a.state is not used

check and remove it

Reply:
Removed it. Working fine.